src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -65,7 +73,15 @@ export default class GroupComparisonPage extends React.Component<
     @observable.ref private store: GroupComparisonStore;
     private queryReaction: IReactionDisposer;
     private urlWrapper: GroupComparisonURLWrapper;
-
+    @observable
+    currentPathway: string = '';
+    @observable
+    rankingChoices: PathwayMapParams = {
+        dropDownTitle: 'Match count',
+        isPercentageMatch: 0,
+        isAlterationEnabled: 0,
+        considerOnlyTCGAPanPathways: true,
+    };
```

Comment:
I think we should move these into a separate class like `GroupComparisonPathwayMapperUserSelectionStore`.

```suggestion
    pathwayMapperUserSelectionStore: GroupComparisonPathwayMapperUserSelectionStore = new GroupComparisonPathwayMapperUserSelectionStore();
```